lib/node_modules/@stdlib/math/generics/random/sample/lib/factory.js

```diff
@@ -259,8 +259,8 @@ function factory() {
 				throw err;
 			}
 		}
-		mutate = opts.mutate !== void 0 ? opts.mutate : conf.mutate;
-		replace = opts.replace !== void 0 ? opts.replace : conf.replace;
+		mutate = ( opts.mutate === void 0 ) ? conf.mutate : opts.mutate;
```

Comment:
This will not work, as `conf.mutate` would then take precedence over `opts.mutate` if the former was falsy.